src/main/java/seedu/address/logic/parser/AddEventParser.java

```diff
@@ -148,4 +158,8 @@ private static boolean isTime(String str) {
         }
     }
 
+    private static boolean isKeywordTo(String str) {
+        return str.equals("to");
+    }
+
```

Comment:
At this point, it's a bit strange that you have a separate method just to call `str.equals()`...